packages/@adobe/spectrum-css-temp/components/toggle/skin.css

```diff
@@ -224,3 +224,76 @@ governing permissions and limitations under the License.
     }
   }
 }
+
+@media (forced-colors: active) {
+  .spectrum-ToggleSwitch {
+    forced-color-adjust: none;
+    --spectrum-switch-emphasized-handle-background-color: ButtonFace;
+    --spectrum-switch-emphasized-handle-border-color: ButtonText;
+    --spectrum-switch-emphasized-handle-border-color-disabled: GrayText;
+    --spectrum-switch-emphasized-handle-border-color-down: Highlight;
+    --spectrum-switch-emphasized-handle-border-color-hover: Highlight;
+    --spectrum-switch-emphasized-handle-border-color-key-focus: ButtonText;
+    --spectrum-switch-emphasized-handle-border-color-selected: Highlight;
+    --spectrum-switch-emphasized-handle-border-color-selected-disabled: GrayText;
+    --spectrum-switch-emphasized-handle-border-color-selected-down: Highlight;
+    --spectrum-switch-emphasized-handle-border-color-selected-hover: Highlight;
+    --spectrum-switch-emphasized-handle-border-color-selected-key-focus: Highlight;
+    --spectrum-switch-emphasized-text-color: CanvasText;
+    --spectrum-switch-emphasized-text-color-disabled: GrayText;
+    --spectrum-switch-emphasized-text-color-down: CanvasText;
+    --spectrum-switch-emphasized-text-color-hover: CanvasText;
+    --spectrum-switch-emphasized-text-color-selected-disabled: GrayText;
+    --spectrum-switch-emphasized-track-color: ButtonFace;
+    --spectrum-switch-emphasized-track-color-disabled: ButtonFace;
+    --spectrum-switch-emphasized-track-color-selected: Highlight;
+    --spectrum-switch-emphasized-track-color-selected-disabled: GrayText;
+    --spectrum-switch-emphasized-track-color-selected-down: Highlight;
+    --spectrum-switch-emphasized-track-color-selected-hover: Highlight;
+    --spectrum-switch-emphasized-track-color-selected-key-focus: Highlight;
+    --spectrum-switch-focus-ring-color-key-focus: ButtonText;
+    --spectrum-switch-handle-border-color: ButtonText;
+    --spectrum-switch-handle-border-color-selected: Highlight;
+    --spectrum-switch-quiet-handle-border-color-selected: Highlight;
+    --spectrum-switch-quiet-handle-border-color-selected-down: Highlight;
+    --spectrum-switch-quiet-handle-border-color-selected-hover: Highlight;
+    --spectrum-switch-quiet-handle-border-color-selected-key-focus: Highlight;
+    --spectrum-switch-quiet-track-color-selected: Highlight;
+    --spectrum-switch-quiet-track-color-selected-down: Highlight;
+    --spectrum-switch-quiet-track-color-selected-hover: Highlight;
+    --spectrum-switch-quiet-track-color-selected-key-focus: Highlight;
+    --spectrum-switch-track-color: ButtonFace;
+    --spectrum-switch-track-color-disabled: ButtonFace;
+    .spectrum-ToggleSwitch-switch {
+      &:after {
```

Comment:
I think `:after` is meant to be the focus ring? With this change, you can see strange behavior when keyboard focusing the switch. It gets larger rather than having a halo focus ring.

Maybe you could do something with a box-shadow on the switch itself rather than a `:before` or `:after`?

```css
 box-shadow: inset 0 0 0 2px  var(--spectrum-switch-handle-border-color);
```

Reply:
great suggestion